src/assets/signal-end-event.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="15" viewBox="0 0 18 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M9 0L18 15H0L9 0Z" fill="#ED4757"/>
+</svg>
```

Comment:
@lisa-fehr and I have been using a rounded version of this symbol 😬maybe once this is merged we can go back and adopt your superior pointy triangle.

Reply:
Aargh not the ~lime juice~ rounded corners